gprofiler/profilers/node.py

```diff
@@ -35,6 +35,17 @@ class NodeDebuggerUnexpectedResponse(Exception):
     pass
 
 
+class NodeDebuggerProcessUndefined(Exception):
+    pass
+
+
+class ExpectedResult:
```

Comment:
I think `ResultType` is a better name for it.

Code like
```
        if expected_result == ExpectedResult.NONE:
            return message
        else:
            raise NodeDebuggerUnexpectedResponse(message) from None
```

which is confusing (what was expected and what was received?) now becomes:
```
        if expected_result == ResultType.NONE:
            return message
        else:
            raise NodeDebuggerUnexpectedResponse(message) from None
```
which is far more clear :)